diff --git a/cmake/GaudiDependencies.cmake b/cmake/GaudiDependencies.cmake
index ee852e817..48af33a40 100644
--- a/cmake/GaudiDependencies.cmake
+++ b/cmake/GaudiDependencies.cmake
@@ -94,9 +94,10 @@ set(THREADS_PREFER_PTHREAD_FLAG YES)
 set(Boost_USE_STATIC_LIBS OFF)
 set(OLD_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS}) # FIXME: the day BoostConfig.cmake handles Boost_USE_STATIC_LIBS correctly
 set(BUILD_SHARED_LIBS ON)
-find_package(Boost 1.70 ${__quiet} CONFIG COMPONENTS system filesystem regex
-  thread python unit_test_framework program_options log log_setup graph)
+find_package(Boost 1.70 ${__quiet}  COMPONENTS system filesystem regex
+  thread python38 unit_test_framework program_options log log_setup graph )
 set_package_properties(Boost PROPERTIES TYPE REQUIRED)
+add_library(Boost::python ALIAS Boost::python38)
 set(BUILD_SHARED_LIBS ${OLD_BUILD_SHARED_LIBS})
 mark_as_advanced(Boost_DIR) # FIXME: the day Boost correctly marks as advanced its variables
 foreach(component IN ITEMS system filesystem regex thread python unit_test_framework

