diff --git a/CMakeLists.txt b/CMakeLists.txt
index cc4b59e2e..960c050f3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,7 +117,7 @@ set(_acts_tbb_version 2020.1)
 # required packages
 # NOTE we use boost::filesystem instead of std::filesystem since the later
 #   is not uniformly supported even on compilers that nominally support C++17 
-find_package(Boost ${_acts_boost_version} REQUIRED CONFIG COMPONENTS filesystem program_options unit_test_framework)
+find_package(Boost ${_acts_boost_version} REQUIRED COMPONENTS filesystem program_options unit_test_framework)
 find_package(Eigen3 ${_acts_eigen3_version} REQUIRED CONFIG)
 
 # the `<project_name>_VERSION` variables set by `setup(... VERSION ...)` have
